src/lib/data/planEntities.ts

```diff
@@ -257,13 +232,12 @@ const getCustomReferenceAndTypeForPlanEntity = ({
     planEntity,
     'entityPrototypeId'
   );
-  const parent = entityAssociations.get(planEntity.id);
   let ref = '';
-  if (parent) {
+  if (planEntity.parentGoverningEntityId) {
     const ge = getRequiredDataByValue(
       governingEntities,
       planEntity,
-      () => parent.parentId
+      () => planEntity.parentGoverningEntityId
```

Comment:
Main purpose of method `getRequiredDataByValue` is to provide a map to lookup as first argument, entity that is used as key during lookup as second argument, and method how to fetch keys of that entity as third argument.

Read the docstring of this method for better explanation. For the third argument, docstring says:
> ... can also use a different key if entity is solely used for producing nice error messages when lookup fails.

That was the case before the changes, but we are now directly depending on a relationship between governing and plan entities, so we should write this as
```suggestion
      (pE) => pE.parentGoverningEntityId
```

In the end, this is essentially the same code functionally, but since this code can be used as model for a newly written code, it could be falsely assumed what the purpose of third argument is.